### PR TITLE
SM-61 login-form

### PR DESCRIPTION
**Pull request by KatarzynaChmielecka**

### CODE REVIEW

File: src/features/loginPage/LoginPage.tsx [hunk 1]
```diff
@@ -1,5 +1,167 @@
...
+    try {
+      const res = await fetch(
+        'https://s-mallets-backend.vercel.app/auth/login',
+        options,
+      );
+
+      if (res.status !== 200) {
+        toast.error(t('toastBad'), {
+          id: toastId,
+        });
```
Comment (vieraboschkova) [inline src/features/loginPage/LoginPage.tsx:44]: you can add a return to exit the function

File: src/features/loginPage/LoginPage.tsx [hunk 2]
```diff
@@ -1,5 +1,167 @@
...
+        navigate('/user');
+      }
+    } catch (error) {
+      toast.error(t('toastBad'));
+    }
+  };
+
+  const logoutHandler = () => {
+    dispatch(authActions.logout());
```
Comment (vieraboschkova) [inline src/features/loginPage/LoginPage.tsx]: You shuold also probably clean user data in the store after logout

File: src/features/loginPage/LoginPage.tsx [hunk 3]
```diff
@@ -1,5 +1,167 @@
...
+  return (
+    <div className="mx-auto mt-8 mb-16 flex w-full max-w-7xl flex-col sm:px-3 md:flex-row md:gap-20 md:px-6 lg:px-8 ">
+      {isAuth && (
+        <>
+          <h1 className="fixed top-0 left-0 z-50">zalogowany</h1>
+          <button onClick={logoutHandler} className="fixed top-10 left-0 z-50">
+            Wyloguj
+          </button>
+        </>
+      )}
```
Comment (vieraboschkova) [inline src/features/loginPage/LoginPage.tsx]: Should probably create an issue to fix that later

File: src/features/loginPage/LoginPage.tsx [hunk 4]
```diff
@@ -1,5 +1,167 @@
...
+  });
+
+  const loginHandler = async (data: object) => {
+    const toastId = toast.loading('Loading...');
+    const options = {
+      method: 'POST',
+      body: JSON.stringify(data),
+      headers: { 'Content-Type': 'application/json' },
+    };
+
```
Comment (vieraboschkova) [inline src/features/loginPage/LoginPage.tsx:34]: I guess if the logout will be somewhere else, you could check on this page if the user is logged in, then riderct him somewhere else or show him a different message and redirect

File: src/features/loginPage/LoginPage.tsx (hunk 1 above)
Comment (KatarzynaChmielecka) [reply src/features/loginPage/LoginPage.tsx:44]: done

File: src/features/loginPage/LoginPage.tsx (hunk 2 above)
Comment (KatarzynaChmielecka) [reply src/features/loginPage/LoginPage.tsx]: Probably it's not exactly my scope :P but i'll change a bit userSlice also ;)

File: src/features/loginPage/LoginPage.tsx (hunk 3 above)
Comment (KatarzynaChmielecka) [reply src/features/loginPage/LoginPage.tsx]: What do you mean? It's only mock to test :)

File: src/features/loginPage/LoginPage.tsx (hunk 4 above)
Comment (KatarzynaChmielecka) [reply src/features/loginPage/LoginPage.tsx:34]: i think it's thing to discuss with others :)

File: src/features/loginPage/LoginPage.tsx (hunk 2 above)
Comment (KatarzynaChmielecka) [reply src/features/loginPage/LoginPage.tsx]: done ;P

File: src/features/loginPage/LoginPage.tsx (hunk 4 above)
Comment (KatarzynaChmielecka) [reply src/features/loginPage/LoginPage.tsx:34]: done ;)

**Review by n0macx — APPROVED** [review]
Good Job!

**Pull request merged**